Add note about GRPC_TRACE (#2043)
stackrox/collector: docs/troubleshooting.md

The runtime configuration can be obtained using
$ curl "http://<collector>:8080/state/runtime-config"

## Troubleshooting GRPC channel

> I case of network connectivity issues it can be hard to figure out the reason
> why Collector refuses to connect to Sensor. The reason for that is use
> WaitForConnected GRPC method, which doesn't betray much information. A
> workaround to get more information is to add `GRPC_TRACE='*'` environment
> variable to the DaemonSet, which will ask the GRPC library to trace all if it's

Reviewer: ```suggestion
In case of network connectivity issues it can be hard to figure out the reason
why Collector refuses to connect to Sensor. The reason for that is the
WaitForConnected GRPC method, which doesn't convey much information. A
workaround to get more information is to add the `GRPC_TRACE='*'` environment
variable to the DaemonSet, which will ask the GRPC library to trace all of its
```
